docs/architecture/adr-052-tendermint-mode.md

```diff
@@ -0,0 +1,73 @@
+# ADR 052: Tendermint Mode
+
+## Changelog
+
+* 27-11-2019: Initial draft from ADR-051
+* 13-01-2020: Separate ADR Tendermint Mode from ADR-051
+
+## Context
+
+- Fullnode mode : fullnode mode does not have any capability to participate on consensus
+- Validator mode : this mode is exactly same as existing state machine behavior. sync without voting on consensus, and participate consensus when fully synced
+- Seed mode : lightweight seed mode maintaining an address book, p2p like [TenderSeed](https://gitlab.com/polychainlabs/tenderseed)
+
+## Decision
+
+We would like to suggest a simple Tendermint mode abstraction
+
+- Which reactor, component to include for each node
+    - fullnode *(default)*
+        - switch, transport
+        - all reactors
```

Comment:
mempool are considered reactors to my knowledge, maybe better as
```suggestion
        - reactors
          - mempool
          - consensus
          - ...
```
then listing all the reactors involved